### PR TITLE
[Graph tuner]Add opt out operator for has_multiple_inputs for graph tuner

### PR DESCRIPTION
**Pull request by zhen-jia**
If layout_transform is in the graph, graph tuner’s has_multiple_inputs will give wrong answer. This PR is to solve this problem.
@kevinthesun please help to review this

### CODE REVIEW

File: python/tvm/autotvm/graph_tuner/dynamic_programming_tuner.py [hunk 1]
```diff
@@ -57,6 +59,8 @@ def __init__(self, *args, **kwargs):
             "input_shapes": self._input_shapes,
             "layout_transform_interlayer_cost": self._layout_transform_interlayer_cost
         }
+        self._opt_out_op = OPT_OUT_OP
```
Comment (kevinthesun) [inline python/tvm/autotvm/graph_tuner/dynamic_programming_tuner.py]: We don't need to do this again for sub class.

Comment (zhen-jia) [reply python/tvm/autotvm/graph_tuner/dynamic_programming_tuner.py]: I see. I will remove that.

**Comment by kevinthesun** [comment]
@zhen-jia Overall lgtm. Can you take a look at the ci failure?

**Comment by zhen-jia** [comment]
> @zhen-jia Overall lgtm. Can you take a look at the ci failure?
I am working on it and fixed the failures on my part. The current failure reported by rust test. It does not seem like caused by my changes. Any ideas?  @kevinthesun

**Comment by kevinthesun** [comment]
You can create a dummy commit and re-trigger ci.

**Comment by kevinthesun** [comment]
Thanks @zhen-jia

**Pull request merged**